experimental/PlaneCurve/ParaPlaneCurves.jl

```diff
@@ -97,9 +141,22 @@ function parametrization_conic(C::ProjPlaneCurve{fmpq})
 end
 
 @doc Markdown.doc"""
-    map_to_rational_normal_curve(C::ProjPlaneCurve)
+    map_to_rational_normal_curve(C::ProjPlaneCurve{fmpq})
+
+Return a rational normal curve of degree $\deg C-2$ which `C` is mapped.
```

Comment:
I don't understand this sentence. Is a word missing?
```suggestion
Return a rational normal curve of degree $\deg C-2$ to which `C` is mapped.
```

Also, I'd expect a function with such a name to return a map, not the image/range of that map? (Of course that goes beyond your PR, as the function is already there and behaves this way, so I am not asking you to change it now, but perhaps this should be considered for a future edit?)